config/manager/metrics_service.yaml

```diff
@@ -12,4 +12,4 @@ spec:
     port: 8080
     targetPort: metrics
   selector:
-    control-plane: controller-manager
+    app: limitador-operator
```

Comment:
are we ok with replacing instead of adding?

Reply:
Yes from my understanding the only thing that's using the service is the service monitor for Prometheus which has also been updated. That being said i don't know all the ins and out of limitador but from what i can see should be good

Reply:
I would rather leave the `control-plane: controller-manager` label. It is a well known label in the k8s controllers world. All the resources related to the operator have the same label, namespaces, services, pods and deployments.

Reply:
Sure let me add it back in